src/Symfony/Component/HttpClient/EventSource.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+namespace Symfony\Component\HttpClient;
+
+use Symfony\Component\EventDispatcher\EventDispatcher;
+use Symfony\Component\HttpClient\Exception\ClientException;
+use Symfony\Component\HttpClient\Exception\InvalidArgumentException;
+
+final class EventSource extends EventDispatcher
```

Comment:
Should we extend the EventDispatcher or wrap it ? I would vote for using composition rather than inheritance.